app/models/location.rb

```diff
@@ -40,7 +40,7 @@ def num_checkins
 
   def completed?
     num_teams = Team.playing.count
-    num_checkins == num_teams
+    num_checkins >= num_teams
```

Comment:
If a team quits partway through the game, it's nice to be able to deactivate them, so we can get an accurate count of who's been to a checkin. But that's a valid point, if the newly-deactivated team has been to a place, and a still-playing team hasn't, deactivating them isn't going to help the count (which would still count as finished because number of teams checked in would == number of teams playing.) But if more teams have checked in than are playing, it counts as not finished, which is the point of this change. 

So, I also need to visit how teams are counted.